shared/chat/conversation/info-panel/index.js

```diff
@@ -21,6 +21,7 @@ import Participants from './participants'
 const border = `1px solid ${globalColors.black_05}`
 const scrollViewStyle = {
   flex: 1,
+  marginTop: -1,
```

Comment:
Ah, this was here earlier and I thought it was vestigial so I removed it. The InfoPanel is already absolutely positioned (see https://github.com/keybase/client/blob/master/shared/chat/conversation/index.desktop.js#L169 ) so you could just change `top: 35` there to `top: 34`, unless there's a reason to keep it that way there and adjust it here.

In any case, though, this attribute only applies for desktop, so it should be put below `borderRight:` on line 30.

Reply:
Ah! Yeah we don't want to see the grey divider. And `top` would just move the whole pane 1px up, which we don't want (there would be 1px missing at the bottom). `marginTop` adds 1px at the top.

Reply:
Good catch, moving to line 30!

Reply:
Ah, ok now I see! Can you add a comment about it?

Reply:
Done!